src/sage/combinat/posets/linear_extensions.py

```diff
@@ -40,7 +40,7 @@
 
 
 class LinearExtensionOfPoset(ClonableArray,
-        metaclass=InheritComparisonClasscallMetaclass):
+                             metaclass=InheritComparisonClasscallMetaclass):
```

Comment:
looks like empty space there? can it be removed?

Reply:
Ah, OK, no problem, sorry.

Reply:
yes, I will remove it, please tell if there are any more edits for it.

Reply:
never mind, it's OK